pii_team_frost.py

```diff
@@ -22,7 +23,8 @@ def find_amex(text) -> list:
 def find_us_ssn(text) -> list:
     """Finds all occurrences of a US social security number in a text string"""
     # match a 9 digit social security number
-    return []
+    MO = re.findall(r'\d{3}-\d{2}-\d{4}', text)
+    return [MO]
```

Comment:
This is no longer a match object - what you were calling MO - so your variable name probably shouldn't be MO.  You are still trying to return that as a list.  I would
```
return re.findall(r'\d{3}-\d{2}-\d{4}', text)
```